### DISCUSSION TITLE
Can the function of adding custom traffic light time and vehicle release direction be added？

### DISCUSSION THREAD

**Discussion by sacefgop** [slyh/Cities2-TrafficLightsEnhancement#113] [General]
Hello mods creator
May I ask if you can add the function of allowing players to freely control the time and direction of traffic lights in the TLE mods? For example, player can freely choose which direction or directions vehicles can pass through, as well as determine the time they can pass. I think the game really needs the function of determining the traffic light time by oneself now. Do you think it can be implemented? Thank you!

**Comment by slyh** [comment]
Please see #111
